config/endpoints/api_entreprise/5_inpi_rne_beneficiaires.yml

```
@@ -0,0 +1,42 @@
+---
+- uid: 'inpi/rne/beneficiaires_effectifs'
+  path: '/v3/inpi/rne/unites_legales/{siren}/beneficiaires_effectifs'
+  position: 300
+  perimeter:
+    entity_type_description: |+
+      Toutes les entreprises inscrit au RCS (Registre du Commerce et des Sociétés) ainsi qu'au RNM (Répertoire Nationale des Métiers)
+    geographical_scope_description:  |+
+      - ✅ France métropolitaine
+      - ✅ DROM COM
+    updating_rules_description: |+
+      Les données sont mises à jour quotidiennement.
+    know_more_description: |+
+      La documentation relative à l'API RNE se trouve sur le lien [suivant](https://data.inpi.fr/content/editorial/Acces_API_Entreprises)
+
+    entities:
+      - entreprises
```

Suite à mes commentaires plus haut, je pense qu'il faut ajouter les associations

```suggestion
      - entreprises
      - associations
```

---

775672272
775688732
775633308
840599930
752610147
451114383
503365801
775639073
777555855
775661820
784314072
784646689
824581623
483197000
780707238
321816944
828197046
788355022
402170765

---

Ok cf https://linear.app/pole-api/issue/API-529#comment-d8331c4a